tl/cli/context-match.py

```diff
@@ -30,9 +32,14 @@ def add_arguments(parser):
                         help='The minimum threshold for similarity with input context for quantity matching. '
                              'Default: 0.85')
     parser.add_argument('--custom-context-file', type=str, dest='custom_context_file', required=False,
-                        help="The file is used to look up context values for matching.")
-    parser.add_argument('--string-separator', action='store', type=str, dest='string_separator', default=",",
-                        help="Any separators to separate from in the context substrings.")
+                        help="The file is used to look up context values for matching.") 
+    parser.add_argument('--string-separator', action = 'store', type=str, dest = 'string_separator', default = ",", 
+                        help = "Any separators to separate from in the context substrings.")
+    parser.add_argument('--use-cpus', action='store', type=int,
```

Comment:
what is the default value? Any time required=False is specified, it should have a reasonable default value